### ISSUE TITLE
Converting Xlsx to CSV, comma being added to the end

### ISSUE THREAD

**Issue by RRobbins94** [EPPlusSoftware/EPPlus#167]
I've got the following xlsx file:

Header1 | Header2 | Header3
1 | 2 | 3
1 | 2 | 3

When trying to convert into a csv its adding a comma at the end of Header3
![image](https://user-images.githubusercontent.com/31695796/90014071-5ce9ef80-dc9e-11ea-9ba4-7b372b092146.png)

Code:

```
                var byteArray = File.ReadAllBytes(path);
                List<string> csvList = new List<string>();
                using (MemoryStream memStream = new MemoryStream(byteArray))
                {
                    ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
                    ExcelPackage package = new ExcelPackage(memStream);

                    var pWorksheet = package.Workbook.Worksheets[0];
                    ExcelWorksheet worksheet = pWorksheet;

                    ExcelRange range = worksheet.Cells[fromRow, 1, worksheet.Dimension.End.Row,
                        worksheet.Dimension.End.Column];

                    worksheet.Tables.Add(range, "Sheet1");
                    var tbl = worksheet.Tables[0];

                    var format = new ExcelOutputTextFormat
                    {
                        Culture = new CultureInfo("en-GB"),
                        Encoding = new UTF8Encoding(),
                    };
                    var csv = worksheet.Cells[tbl.Address.Address].ToText(format);
                    csvList.Add(csv);
                    
                    return csvList;
```

**Comment by swmal** [comment]
Hi,

this has been fixed and will be released in the next version (hopefully next week), see issue #154 . You can try it out with our development build nuget feed https://ci.appveyor.com/nuget/epplus

/Mats

**Comment by JanKallman** [comment]
Fixed in EPPlus 5.3
